CODE_OF_CONDUCT.md

```diff
@@ -53,29 +53,23 @@ If a community member engages in unacceptable behavior, the community organizers
 
 ## 6. Reporting Guidelines
 
-If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. [CONTACT_INFO_HERE].
-
-[LINK_TO_REPORTING_GUIDELINES]
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. Community organizers can be found on the [#kafka-go](https://gophers.slack.com/archives/CG4H0N9PX).
```

Comment:
I think we might be missing something here:
```
Community organizers can be found on the [#kafka-go](https://gophers.slack.com/archives/CG4H0N9PX)
```

"on the" could also become "at" like you have later in the document.  Otherwise maybe we could append "Slack channel"?